cmd/v/help/doc.txt

```diff
@@ -1,16 +1,21 @@
 Usage: 
-  v doc [flags] module_name output_file[.json|.md|.html]
-  v doc [flags] path_to_source.v output_file[.json|.md|.html]
-  v doc [flags] ./directory output_file[.json|.md|.html]
-  v doc [flags] . output_file[.json|.md|.html]
+  v doc [flags] [module_name / folder / V file]
+
+Examples:
+  v doc os
+  v doc -o math.html math
+  v doc -m -f html vlib/
 
 Generates the documentation of a given directory, module, or V source file 
 and prints or saves them to its desired format. It can generate HTML, JSON, 
 or Markdown format.
 
 Options:
   -all    Includes private and public functions/methods/structs/consts/enums.
+  -f      Specifies the output format to be used.
   -loc    Show the locations of the generated signatures. (For plaintext only)
```

Comment:
idk, loc could mean lines of code too. Maybe we find another word?

Reply:
Maybe `-l`? `-location` or `-show-location` is too long.

Reply:
`-l` is fine imo.